pt_cli/connect.py

```diff
@@ -16,12 +16,18 @@
 class Error(Exception):
     """docstring for Error"""
     def __init__(self, msg):
-        self.args = (f"{type(self).__name__}: {msg}",)
+        if isinstance(msg, list):
+            self.args = (f"{type(self).__name__}: \n{chr(10).join(msg)}",)
+        else:
+            self.args = (f"{type(self).__name__}: {msg}",)
         sys.exit(self)
 
 class BadRequestError(Error):
     """docstring for BadRequestError"""
 
+class BadRequestWarning(Error):
```

Comment:
Not whure what you are doing with that, but I guess it should be a subclass of Warning, not Error...
https://docs.python.org/3/library/warnings.html#warning-categories